app/controllers/admin/documentations_controller.rb

```diff
@@ -1,6 +1,8 @@
 class Admin::DocumentationsController < Admin::AdminController
   include TranslationSaveable
 
+  before_action :find_documentation, only: [:edit, :show]
```

Comment:
`update` should probably be in this list too

Reply:
make sure you remove the other find method from update

Reply:
for some reason, this was breaking the page
will take another look

Reply:
maybe try doing `find(params[:html_identifier])` for all of them instead then, but they should all share the same find action one way or another.